.tools/bin/update-root-composer

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env php
+<?php
+
+$replace = [];
+
+$baseDir = dirname(__DIR__, 2);
+$dirs = [
```

Comment:
```suggestion
// list of core addons with a composer.lock.
// intentionally not using a wildcard because other non-system-addons might be contained
$dirs = [
```